lms/djangoapps/commerce/models.py

```diff
@@ -33,15 +37,35 @@ class Meta(object):
             'Specified in seconds. Enable caching by setting this to a value greater than 0.'
         )
     )
-    receipt_page = models.CharField(
-        max_length=255,
-        default='/commerce/checkout/receipt/?orderNum=',
-        help_text=_('Path to order receipt page.')
+    site = models.ForeignKey(
+        Site,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        related_name='commerceconfiguration'
```

Comment:
What is the default related_name here?  If it is "commerceconfiguration" then you can drop this attribute altogether.